EventFilter/L1TXRawToDigi/plugins/L1TCaloLayer1RawToDigi.cc

```diff
@@ -168,7 +171,10 @@ L1TCaloLayer1RawToDigi::produce(Event& iEvent, const EventSetup& iSetup)
       const uint64_t *fedRawDataArray = (const uint64_t *) fedRawData.data();
 
       if ( fedRawData.size() == 0 || fedRawDataArray == nullptr ) {
-        LogError("L1TCaloLayer1RawToDigi") << "Could not load FED data for " << fed << ", putting empty collections!";
+        if (warnsa<5) {
```

Comment:
Note that this will print the LonInfo 5 times per stream, not just 5 times per job.

@Dr15Jones is there a way to set the default limit on the MessageLogger from within the C++ code ?
In the full process configuration one can do
```python
process.MessageLogger.categories.append("L1TCaloLayer1RawToDigi")
process.MessageLogger.L1TCaloLayer1RawToDigi = cms.untracked.PSet(
    limit = cms.untracked.int32( 5 )
)
```
Is there a way to do the same in C++ within the module that will print the messages ?
Something like
```c++
if (edm::Service<MessageLogger>().isAvailable())
  edm::Service<MessageLogger>()->category("L1TCaloLayer1RawToDigi").setDefaultLimit(5);
```
?